plugin/core/windows.py

```diff
@@ -437,6 +440,34 @@ def get_project_path(self, file_path: str) -> Optional[str]:
                     candidate = folder
         return candidate
 
+    def should_present_diagnostics(self, uri: DocumentUri) -> Optional[str]:
+        scheme, path = parse_uri(uri)
+        if scheme != "file":
+            return None
+        if not self._workspace.contains(path):
+            return "not inside window folders"
+        view = self._window.active_view()
+        if not view:
+            return None
+        settings = view.settings()
+        if self._matches_glob(path, settings.get("binary_file_patterns")):
```

Comment:
I have this in my project:

![Screenshot 2021-08-01 at 10 24 10](https://user-images.githubusercontent.com/153197/127764531-5bb77fc3-6d90-4f73-8c99-a88c72cab42e.png)

and this are the view settings (the view is within the workspace):

```
>>> view.settings().get('folder_exclude_patterns')
['.svn', '.git', '.hg', 'CVS', '.mypy_cache', 'node_modules']
```

It looks like the `.nuxt` is missing (`node_modules` comes from global override). So are the view settings not actually updated with project overrides?

Reply:
Hmm, I can reproduce this behavior here as well. Seems broken to me.

Reply:
https://github.com/sublimehq/sublime_text/issues/4717